### PR TITLE
Update CITATION.cff

### PR DESCRIPTION
**Pull request by PedanticHacker**
Fixed the `message` text so that it is written on one line only. Also, fixed the `abstract` text to resemble the first line of the `help` command output in a CLI (as was authored by me). Furthermore, rearranged the keywords from the shortest to the longest. And fixed the naming of the key `name` to the perhaps more correct naming `given-names` as per CFF format guide.

### CODE REVIEW

**Comment by vondele** [comment]
concerning name vs given-names see the discussion here https://github.com/official-stockfish/Stockfish/pull/4422#discussion_r1125634378

**Comment by PedanticHacker** [comment]
> concerning name vs given-names see the discussion here [#4422 (comment)](https://github.com/official-stockfish/Stockfish/pull/4422#discussion_r1125634378)

I was surprised when I parsed the original `CITATION.cff` file and the error I got was `Property 'name: The Stockfish developers (see AUTHORS file)' inside 'authors' was ignored. Check if the key is correct.`.

You can try it yourself here: https://citation-file-format.github.io/cff-initializer-javascript/#/update. Notice that if you change the naming of the key `name` to `given-names`, the error mesage goes away after the new parsing.

@vondele, please instruct me what to do.

File: CITATION.cff [hunk 1]
```diff
@@ -4,11 +4,11 @@
 cff-version: 1.2.0
 title: Stockfish
 message: >-
-  If you use this software, please cite it using the
+  If you use cffinit, please cite it using the
```
Comment (vondele) [inline CITATION.cff:7]: ?

Comment (PedanticHacker) [reply CITATION.cff:7]: Oops! I ment to say _Stockfish_ here. Thanks for pointing out my mistake, @vondele.

Comment (PedanticHacker) [reply CITATION.cff]: It is fixed now in the latest commit.

**Pull request closed without merging**